src/main/java/seedu/address/logic/parser/DateCommandParser.java

```diff
@@ -21,15 +27,60 @@ public class DateCommandParser implements Parser<DateCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public DateCommand parse(String args) throws ParseException {
-        requireNonNull(args);
-        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_DATE);
-        Index index;
         try {
-            index = ParserUtil.parseIndex(argMultimap.getPreamble());
+            requireNonNull(args);
+            ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args,
+                    PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_DATE);
+            if (!areAnyPrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL)
+                    && !areAnyPrefixesPresent(argMultimap, PREFIX_DATE)) {
+                throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT, DateCommand.MESSAGE_USAGE));
+            }
+
+            Optional<String> name = argMultimap.getValue(PREFIX_NAME);
```

Comment:
The check for command format using areAnyPrefixesPresent is effective; however, ensure that all prefixes are consistently validated to prevent silent failures.